test/vaadin-checkbox-group-material_test.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
```

Comment:
I created the new file because I would use the material version, instead of using the lumo. And I do not want other tests will be affected if I use the material version.

Reply:
Theme-specific changes should rather be tested with visual tests.

Reply:
I added visual tests for its